### PR TITLE
Update clinical trials variables/figures in manuscript

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This PR builds on #858 to switch the clinical trials figures/variables to report only interventional clinical trials, in line with a suggestion from @byrdjb.

### Suggested reviewers (optional)

@agitter 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -92,9 +92,10 @@ As results become available from additional clinical trials, we will continue to
 
 ![
 **COVID-19 clinical trials.**
-There are {{ebm_trials}} COVID-19 clinical trials and {{ebm_trials_results}} trials with results as of {{ebm_date_pretty}}.
-The recruitment statuses and trial phases are shown only for trials in which the status or phase is recorded.
-The study types include only types used in at least five trials.
```
Comment (agitter) [inline content/20.pharmaceuticals.md:97]: Should we keep this note about which study types are shown?

**Review by agitter — CHANGES_REQUESTED** [review]
These changes look great.  We'll have to merge #858 first to test the new template variables in the build.

Can you also please update the caption for the map in Figure 6 so we don't forget to do that later?  It looks like that also uses only interventional trials now.

File: content/20.pharmaceuticals.md (hunk 1 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:97]: Yes, I agree we should! Should be fixed in 235e5e3

**Comment by rando2** [comment]
Good catch! Choropleth caption is also updated now, 677ec39

**Review by agitter — APPROVED** [review]

**Comment by agitter** [comment]
Looks great.  Now that #858 is merged we could try re-running the Appveyor build to make sure the images show up correctly before merging this.

**Comment by rando2** [comment]
Oh no! I was too quick on the merge! I can revert if that's better!

**Comment by agitter** [comment]
Let's leave it.  We can debug images in a new PR if we need to.

**Comment by agitter** [comment]
It looks like the updated interventional figures worked: https://greenelab.github.io/covid19-review/v/ff44f6b8397192b6b47d6332beae03e66f25583d/

**Comment by rando2** [comment]
@agitter I'm actually not sure if that figure was generated using the updated code -- I thought I rearranged the figure panels so that the breakdown of interventional vs other trial types was first (I tried to, at least, I haven't done too much plotting in Python recently since most of my old lab worked in R). Do we need to wait until tomorrow to see these changes in the build?

**Comment by agitter** [comment]
@rando2 the subpanel order wasn't actually changed in #858.  I'll comment there in case you want to make a quick new pull request that reorders the panels.

**Pull request merged**